src/VisualStudio/VisualBasic/Impl/VSPackage.resx

```diff
@@ -194,4 +194,10 @@
     <value>Visual Basic Tools</value>
     <comment>Help &gt; About</comment>
   </data>
+  <data name="Visual_Basic_Script" xml:space="preserve">
+    <value>Visual Basic Script</value>
```

Comment:
So we're removing _interactive_ support but leaving script support? Just want to confirm.

Reply:
Yes

---
In reply to: [261425890](https://github.com/dotnet/roslyn/pull/29069#discussion_r261425890) [](ancestors = 261425890)